db/migrate/20190930164241_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :username
+      t.string :password_digest
```

Comment:
good that it is a digest

Reply:
yes, password is encrypted using that bcrypt gem